yuhow/yuhowNote.tex

```diff
@@ -1,4 +1,5 @@
-\documentclass[a4paper,12pt,dvips]{article}
+%\documentclass[a4paper,12pt,dvips]{article}
+\documentclass[a4paper,12pt]{article}
```

Comment:
I suppose you [are using dvips now](https://github.com/yuhow/cesenote/commit/6470db233dcc2b5a94043feb7771de2e03497d2e#diff-3505381d264d0e9bd5b841f26866559cR50).  In this case it would be clear to revert this change to explicitly specify the driver as it did.